packages/manager/modules/exchange/src/domain/domain.service.js

```diff
@@ -197,6 +197,14 @@ export default class ExchangeDomains {
       .then(({ data }) => data);
   }
 
+  getDkimSelectorName(organization, serviceName, domain, selectorName) {
+    return this.services.$http
+      .get(
+        `/email/exchange/${organization}/service/${serviceName}/domain/${domain}/dkim/${selectorName}`,
```

Comment:
```suggestion
  getDkimSelectorName(organization, serviceName, domain, selector) {
    return this.services.$http
      .get(
        `/email/exchange/${organization}/service/${serviceName}/domain/${domain}/dkim/${selector}`,
```
To be consistent with all other functions